hooks/use-selected-terms-of-saved-post/readme.md

```diff
@@ -0,0 +1,20 @@
+# `useSelectedTermsOfSavedPost`
+
+The `useSelectedTermsOfSavedPost` hook retrieves the ids of the selected terms of the provided taxonomy. It gets used internally by the [`useSelectedTerms`](../use-selected-terms/) hook which is a more full featured version of this hook.
+
+## Usage
+
+```js
+import { useSelectedTermsOfSavedPost } from '@10up/block-components';
+
+function BlockEdit(props) {
+    const { context } = props;
+    const { postId } = context;
+    const [selectedCategoriesOfSavedPost, hasResolvedSelectedCategoriesOfSavedPost] =
+  useSelectedTermsOfSavedPost('category', postId);
```

Comment:
[nit] Indentation issue here

Reply:
Fixed in [5d416d9](https://github.com/10up/block-components/pull/178/commits/5d416d99c0d37deb4a01841b58d328d51567f517)